symplyphysics/laws/relativistic/coordinate_conversion.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve, sqrt)
+from sympy.physics.units import speed_of_light
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+
+# Description
+## Let the frame of reference move relative to the other frame of reference at a constant speed along the X
+## axis, and the origin of the spatial coordinates coincide at the initial moment of time in both systems.
+## Then there are simple transformations that can be used to get the x coordinate in one frame of reference,
+## knowing the x coordinate in another frame of reference.
+
+## Law is: x_2 = (x_1 - v * t) / (1 - (v / c)^2), where
+## x_2 - another coordinate (in the second frame of reference),
+## x_1 - coordinate (in the first frame of reference),
+## v - velocity at which the second frame of reference moves relative to the first,
+## a - time (in the first frame of reference).
+
+another_coordinate = Symbol("another_coordinate", units.length)
+
+coordinate = Symbol("coordinate", units.length)
+velocity = Symbol("velocity", units.velocity)
+time = Symbol("time", units.time)
```

Comment:
It is important to distinguish frame of reference for the user. Add `first_frame`, `second_frame` to symbols.